optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/__init__.py

```diff
@@ -521,6 +522,11 @@ def initialize(self, config):
                                  % (entry))
             elif len(entry) == 2:   # key, entry
                 entry.insert(0, '')  # add path='' (root)
+            # We try to maintain the data type of the provided value
+            try:
+                entry[2] = ast.literal_eval(entry[2])
```

Comment:
My only concern here is the warning the this module documentation:

```


Warning

It is possible to crash the Python interpreter with a sufficiently large/complex string due to stack depth limitations in Python’s AST compiler. 
```

Reply:
@beraldoleal What do you suggest we do here ? Warn the users here ?  I do not think users will want to input such a complex string to `--mux-inject`, but we can still warn the users. There is no potential solution to it, since we can not catch the Segmentation fault, and RecursionError is not raised beyond a limit.

So, I can catch RecursionError, and add a warning. Is there anything else you suggest ?

Reply:
The ideal scenario would be to write our own parser here to avoid using literal_eval. But not sure how complicated it is. Just thinking out of loud. So yes, I'm ok with this.

Reply:
Sure, done. Please check the latest push.